### ISSUE TITLE
Weird looking Output selector menu

### ISSUE THREAD

**Issue by hbons** [microsoft/vscode#208026]
- [ ] 1. The font size of the the menu log selector in the Output view is too small. This makes the text look uncentered vertically.
- [ ] 2. The separator is just a bunch of dashes "---" instead of a native separator. This is not good for accessibility. 
 
It's possible that a smaller font size was chosen to be able to see more items in this relatively long menu. But that doesn't work in this case on macOS as the menu item height is still the same, so there aren't more items displayed than before.

 If a more compact menu is desired, it's better to set the [NSControlSize](https://developer.apple.com/documentation/appkit/nscontrolsize) on macOS, instead of changing the font size. This will actually allow more menu items to be shown at the same time.

Here's a screenshot showing the text misalignment compared to a regular menu (right).

<img width="943" alt="image" src="https://github.com/microsoft/vscode/assets/103326/f23d81d3-e6d9-4932-b2ad-ad3da29a9953">

https://developer.apple.com/documentation/appkit/nscontrolsize

Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.88.0-insider 
- OS Version: macOS

Steps to Reproduce:

1. On macOS, open the Panel and click the Output tab.
2. Notice misaligned text and non-native separator.

**Comment by bpasero** [comment]
This would require changes in https://github.com/electron/electron, so marking as upstream.
